### PR TITLE
small fix to the README.md where the command said to be run had also …

### PR DESCRIPTION
**Pull request by ghost**
…the outut copied confusing whoever tried to copy it(including me)

### CODE REVIEW

**Comment by justwheel** [comment]
Hi @Viliansh, thanks for making a contribution to the project! The line you are removing is the output from the console. My first impression is that it is still useful to leave it there, so that someone who runs the above shell command is able to get the expected output from the command. What do you think? Did you get the same output when you ran the shell command on the top line?

**Comment by ghost** [comment]
> Hi @Viliansh, thanks for making a contribution to the project! The line you are removing is the output from the console. My first impression is that it is still useful to leave it there, so that someone who runs the above shell command is able to get the expected output from the command. What do you think? Did you get the same output when you ran the shell command on the top line?

yep, i get the same output, but the way it's formatted give the impression that everything inside the block was the command(the github also put an copy-button and it copy everything), i'm gonna try to keep the output in a separated block, so it fixes the github copy-button

**Comment by justwheel** [comment]
@Viliansh That seems smart! If you separate them out into two distinct blocks, I'm fine to merge this 👍🏻 Make sure to remove the shell prompt from the copy-paste dialogue too, i.e. `$`

**Comment by ghost** [comment]
just putting it in another code block would made it weird , so i tried to merge it with the next line, what do think?

File: README.md [hunk 1]
```diff
@@ -66,9 +66,8 @@ Create a virtualenv into which you can install the module.
 Run the script with the Fedora configuration::
 
     $ ./asknot-ng.py templates/index.html questions/fedora.yml l10n/fedora/locale --theme fedora
```
Comment (justwheel) [inline README.md]:
```suggestion
./asknot-ng.py templates/index.html questions/fedora.yml l10n/fedora/locale --theme fedora
```

This removes the shebang from the line, which will cause an error in the terminal if pasted in.

**Comment by ghost** [comment]
do you want me to remove the $ from the others commands too?

**Comment by justwheel** [comment]
@Viliansh If we are making this change here, it would be good for the README to be consistent throughout.

**Comment by ghost** [comment]
did it!, what do you think?

**Pull request closed without merging**